scripts/exregional_run_post.sh

```diff
@@ -304,9 +304,9 @@ done
 rm -rf ${DATA_FHR}
 
 # Delete the forecast directory
-if [ $RUN_ENVIR = "nco" ] && [ $KEEPDATA = "FALSE" ]; then
-   rm -rf $DATAFCST
-fi
+# if [ $RUN_ENVIR = "nco" ] && [ $KEEPDATA = "FALSE" ]; then
+#   rm -rf $DATAFCST
+# fi
```

Comment:
This if-statement is necessary to make KEEPDATA:false work correctly.

Reply:
DATAFCST is the DATA for the forecast job. It is wrong solution for a post job to unconditionally remove the forecast job DATA that was used as the input of the post job. NCO will unable to recover, rerun or debug during system emergency.

Reply:
I would agree that having the post script be able to remove the fcst run directory isn't a good practice.  I think the introduction of an external cleanup job/script like is included in this PR makes more sense.